.drone.yml

```diff
@@ -17,6 +17,16 @@ pipeline:
     when:
       event: [pull_request]
 
+  build_solidity_docs:
+    group: build_artifacts
+    image: kowalatech/solidoc
+    commands:
+      - make dev_solidocs_docker_image
```

Comment:
I think we could build this image once and push it to dockerhub instead of building it every time. It could live in a separate repo too.

Reply:
Will do that